hackathon_site/hardware/models.py

```diff
@@ -95,3 +96,22 @@ class Incident(models.Model):
 
     def __str__(self):
         return self.id
+
+
+class AnnotatedHardwareManager(models.Manager):
```

Comment:
This needs to be defined above the Hardware model (I would put it right above it), python can't reference things defined later in the file like JavaScript can